docs/references/faq.md

```diff
@@ -135,3 +135,147 @@ out_batch = pa.Table.from_arrays([new_data, batch.column('ColumnToKeepAsIs')], s
 Use the newest version of the Python SDK which contains fixes that concern date and time. Use the `arrow_scalar_date32_to_py` (`ayx_python_sdk.providers.amp_provider.utilities.conversions`) in case you use the tool in an older version of Designer.
 
 Older versions of Designer process the date incorrectly. This function ensures dates are set correctly for those versions of Designer, otherwise the function simply does nothing.
+
+## _"Why does the CLI use Python?"_
+
+Since we were already developing a Python SDK, it made sense to use
+Python for the CLI, too. Python lets us leverage the [doit automation
+tool](https://pydoit.org/) to build the CLI. We chose to use
+[Typer](https://typer.tiangolo.com/) in combination with doit because it lets us map CLI
+commands to doit tasks.
+
+
+## _"Can I use the CLI and SDK behind a firewall?"_
+
+### _"Do I need to whitelist anything to use the CLI?"_
+
+To download and install the CLI, receive notifications of newer versions of the
+CLI, or check for updates to the CLI, you need to whitelist
+https://pypi.org/.
+Otherwise, feel free to use the CLI completely offline.
+
+### _"What packages do I need to download?"_
+
+All of the required Python packages that you need to get started with
+development are downloaded and installed when you install the CLI and
+SDK (as part of their dependencies).
+
+The requirements for each package are:
+
+-   **SDK**: \"Click==7.1.2\", \"grpcio==1.39.0\", \"numpy\>=1.17.1\",
+    \"deprecation==2.1.0\", \"pyarrow==5.0.0\", \"pandas==1.1.0\",
+    \"protobuf\>=3.5.0.post1\", \"psutil==5.6.3\", \"pydantic==1.8.2\",
+    \"python-dateutil==2.8.1\", \"pytz==2020.1\", \"shiv==0.3.1\",
+    \"six==1.14.0\", \"typer==0.3.1\", \"xmltodict==0.12.0\",
+    \"requests==2.24.0\", PyPAC==0.15.0\"
+-   **CLI**: \"typer==0.3.1\", \"doit==0.33.1\", \"pydantic==1.8.2\",
+    \"packaging==20.4\", \"requests==2.24.0\", \"xmltodict==0.12.0\"
+
+
+## _"Why another Python SDK?"_
+
+The original [Python Engine
+SDK](https://help.alteryx.com/developer-help/python-engine-sdk) is complex, runs in-process, and can be slow. The [AYX
+Python SDK](https://help.alteryx.com/developer-help/ayx-python-sdk) design solves these issues. It runs
+out-of-process using gRPC to communicate. It also simplifies the
+development process of new tools to 4 function calls, abstracting away a
+lot of the previous SDK function calls.
+
+By running the plugin out-of-process, we are able to efficiently manage
+multiple plugins and are not bound by the restrictions of the C++
+`boost` library.
+
+
+## _"Python Engine SDK and AYX Python SDK. What's the difference?"_
+
+[AYX Python SDK](https://help.alteryx.com/developer-help/ayx-python-sdk-v2) is a new SDK product. It improves upon
```

Comment:
Point this to the new reference on ayx-developer-sdk repo.

This should point to the corresponding markdown version